packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -41,6 +42,8 @@ exchangePassRouterPackage = "reya-exchange-pass-nft:1.0.0@router"
 ranksRouterPackage = "reya-ranks:1.0.0@router"
 ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.3@router"
+#oracleAdaptersProxyPackage = "reya-oracle-adapters:1.0.0@proxy"
```

Comment:
update to 1.0.1

Reply:
I've actually got rid of mainnet for now, we'll push this when doing the mainnet deployment